### PR TITLE
RMET-2623 ::: Fix nil optionsAllowed for Blood Glucose

### PR DESCRIPTION
**Pull request by OS-ricardomoreirasilva**
## Description
The `HealthKitVariable` object associated with "Blood Glucose" is missing a `optionsAllowed` value. This was throwing a unhandled exception that caused an app crash when trying to use Blood Glucose with a disallowed operation.

Two Sample App builds were created:
- One with MABS 7.2: http://engineering.outsystems.net/MABS/BuildDetail.aspx?BuildId=baf8214dacf6674ae1fcc12f988b137913c9fb09&StageId=3
- Another with MABS 8.1: http://engineering.outsystems.net/MABS/BuildDetail.aspx?BuildId=9fd1c9f286a9e7af050e8b06db2f903bd371f835&StageId=3

## Context
https://outsystemsrd.atlassian.net/browse/RPM-2623

## Type of changes
- [X] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [X] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [ ] Android
- [X] iOS
- [ ] JavaScript

## Checklist
- [X] Pull request title follows the format `RMET-XXXX <title>`
- [X] Code follows code style of this project
- [X] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

**Review by CarlsCorrea — COMMENTED** [review]
I forgot to mention, the xcframework needs to be built using Xcode 12 to work with MABS 7. Otherwise we have an error regarding the version of the compiler. I think @nflsilva can help us on that.

**Comment by OS-ricardomoreirasilva** [comment]
> I forgot to mention, the xcframework needs to be built using Xcode 12 to work with MABS 7. Otherwise we have an error regarding the version of the compiler. I think @nflsilva can help us on that.

I know what to do ;) Should I use a specific xCode 12 version or is the latest ok?

File: plugin.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<plugin id="com.outsystems.plugins.healthfitness" version="1.2.2" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
+<plugin id="com.outsystems.plugins.healthfitness" version="1.2.4" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (alexgerardojacinto) [inline plugin.xml:2]: Shouldn't it be version 1.2.3?

**Review by alexgerardojacinto — CHANGES_REQUESTED** [review]

**Comment by CarlsCorrea** [comment]
> > I forgot to mention, the xcframework needs to be built using Xcode 12 to work with MABS 7. Otherwise we have an error regarding the version of the compiler. I think @nflsilva can help us on that.
> 
> I know what to do ;) Should I use a specific xCode 12 version or is the latest ok?

ok, yes can be the latest.

File: plugin.xml (hunk 1 above)
Comment (alexgerardojacinto) [reply plugin.xml:2]: Ohhh nevermind, makes sense.

**Review by alexgerardojacinto — APPROVED** [review]

**Comment by OS-ricardomoreirasilva** [comment]
> > > I forgot to mention, the xcframework needs to be built using Xcode 12 to work with MABS 7. Otherwise we have an error regarding the version of the compiler. I think @nflsilva can help us on that.
> > 
> > 
> > I know what to do ;) Should I use a specific xCode 12 version or is the latest ok?
> 
> ok, yes can be the latest.

Fixed it. It's now building with both MABS 7.2 and 8.1.

image

**Review by CarlsCorrea — APPROVED** [review]

**Pull request merged**